core/management/commands/ethnicities_with_issues.py

```diff
@@ -7,4 +9,4 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         for e in models.Ethnicity.objects.all():
-            print e.name, ": ", e.has_issues
+            print(e.name, ": ", e.has_issues)
```

Comment:
could this use logging instead of print? maybe there is a good reason

Reply:
I'm assuming it was just habit but since that's a management command it probably should be using `self.stdout` / `self.stderr`